src/Symfony/Component/Notifier/Bridge/Mercure/MercureTransport.php

```diff
@@ -11,7 +11,9 @@
 
 namespace Symfony\Component\Notifier\Bridge\Mercure;
 
-use Symfony\Component\Mercure\PublisherInterface;
+use Symfony\Component\Mercure\Exception\InvalidArgumentException as MercureInvalidArgumentException;
+use Symfony\Component\Mercure\Exception\RuntimeException as MercureRuntimeException;
```

Comment:
no need for the aliases

Reply:
Aliases are here to avoid collisions which Notifier's exceptions. Maybe you prefer to specify the whole FQCN?

Reply:
ah, in that case it's okay.